### ISSUE TITLE
Hi! I have the same error updating from 2.4.1 to 2.4.2.

### ISSUE THREAD

**Issue by kekegt** [FaserF/hassio-addons#82]
Hi! I have the same error updating from 2.4.1 to 2.4.2.

what device (hardware)
Generic x86-64

What DNS server are you using (Internet provider, google, cloudflare, PiHole, Adguard?)
Adguard (Tried with protection off also)
Google DNS when protection is off

Any VPN used?
Nope

What Homeassistant OS version
2022.8.0

Your country
Canada

Here is the error:
----------------------------------------------------------------------
```
Logger: homeassistant.core
Source: components/hassio/update.py:169
First occurred: 9:06:34 PM (3 occurrences)
Last logged: 9:22:43 PM

Error executing service: <ServiceCall update.install (c:01G9K65RN90GR2YMKR8EB3TNNF): entity_id=['update.apache2_update'], backup=True>
Error executing service: <ServiceCall update.install (c:01G9K6FCT0EBBV6GMXN1P2H5RK): entity_id=['update.apache2_update'], backup=True>
Error executing service: <ServiceCall update.install (c:01G9K73EYSN072VMFW5G6334A4): entity_id=['update.apache2_update'], backup=True>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/hassio/update.py", line 167, in async_install
    await async_update_addon(self.hass, slug=self._addon_slug, backup=backup)
  File "/usr/src/homeassistant/homeassistant/components/hassio/handler.py", line 45, in _wrapper
    raise HassioAPIError(data["message"])
homeassistant.components.hassio.handler.HassioAPIError: The command '/bin/ash -o pipefail -c apk --no-cache add mosquito mosquito-dev' returned a non-zero code: 2

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/core.py", line 1731, in catch_exceptions
    await coro_or_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1750, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 204, in handle_service
    await service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 676, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 930, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 713, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/update/__init__.py", line 152, in async_install
    await entity.async_install_with_progress(version, backup)
  File "/usr/src/homeassistant/homeassistant/components/update/__init__.py", line 411, in async_install_with_progress
    await self.async_install(version, backup)
  File "/usr/src/homeassistant/homeassistant/components/hassio/update.py", line 169, in async_install
    raise HomeAssistantError(f"Error updating {self.title}: {err}") from err
homeassistant.exceptions.HomeAssistantError: Error updating Apache2: The command '/bin/ash -o pipefail -c apk --no-cache add mosquito mosquito-dev' returned a non-zero code: 2
```
----------------------------------------------------------------------

_Originally posted by @kekegt in https://github.com/FaserF/hassio-addons/issues/33#issuecomment-1204658391_

**Comment by FaserF** [comment]
Are you running Homeassistant OS or just Homeassistant as a container? 
Could you please try to install any other of my addons except the other apache2 addons and let me know if they are working for you?

**Comment by kekegt** [comment]
I'm running Home assistant OS. 
I tried installing Bash Script Executer and it worked. 
Please let me know if you need any other details.

**Comment by stevnhoved** [comment]
I have a similar error.
I try to upgrade fra version 2.3.0 to 2.4.2 and get this error:

> The command '/bin/ash -o pipefail -c apk --no-cache add apache2 libxml2-dev apache2-utils apache2-mod-wsgi apache2-ssl' returned a non-zero code: 5

I am running Home assistant OS on a Raspberry Pi 4
I have tried to uninstall the add-in and install it from scratch, but gets the same error.

I am using my ISP DNS and have the same problem on HA 2022.7x and 2022.8.x

**Comment by FaserF** [comment]
issue should be fixed with Version 2.4.3
Thanks for reporting the issue! This one was a different one than the one in issue 33.

**Comment by kekegt** [comment]
Hi! Happy to report that the update process worked with 2.4.3!
